schema_utils.py

```diff
@@ -426,6 +426,18 @@ def is_valid_email(obj):
         """
         return bool(re.search(r'^[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}$', obj))
 
+    @staticmethod
+    def is_valid_ratio(obj):
+        """Ensures that `obj` (a string) is a valid ratio input.
+
+        Args:
+            obj: str. A string.
+
+        Returns:
+            bool. Whether the given object is a valid ratio input.
+        """
+        return bool(re.search(r'^\d+(:\d+)+$', obj))
```

Comment:
Needs to factor in spaces. ``   1 : 2 `` should be a valid ratio.

Reply:
not valid anymore